### PR TITLE
Angular directives

### PR DESCRIPTION
**Pull request by piyapan039285**

### CODE REVIEW

File: angular/directives/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+})
+export class HighlightDirective {
+
+  constructor(private el: ElementRef) { }
+
+  @Input('appHighlight') highlightColor: string;
+
+  ngOnChanges(changes: SimpleChanges){
+    // set background color.
+    if(changes.highlightColor){
```
Comment (earthpyy) [inline angular/directives/README.md]:
```suggestion
    if (changes.highlightColor){
```

File: angular/directives/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Angular directives is custom field in HTML. It will add custom logic to HTML elements. For example, changing CSS or HTML contents.
+
+## Creating Directives Manually
+
+Step below will guide you by example.
+
+### Goal
+
+You want to create custom field `appHighlight` which will add background color in the HTML element.
```
Comment (earthpyy) [inline angular/directives/README.md:11]: Can we add the instruction about naming a new directive krub? For example, we should prefix them with `app` (need to followed from `angular.json`).

**Review by earthpyy — COMMENTED** [review]
Only few things krab

**Review by earthpyy — APPROVED** [review]
Thanks krubb.

**Review by NorakGithub — APPROVED** [review]

**Pull request merged**